ansible/roles/install_python/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+- name: Installing the python3.11-venv package
+  become: true
+
+  block:
+    - name: python3.11-venv package
+      apt:
+        name: python3.11-venv
+        state: present
+      
+    - name: Creating a virtual environment
+      command: python3.11 -m venv /home/authz/venv
+      args:
+        chdir: /home/authz/
+
+    - name: Change ownership of the virtual environment directory
+      file:
+        path: /home/authz/venv
+        owner: authz
+        group: authz
+        mode: "0750"
```

Comment:
I think this changes permissions on global python interpreter
```
[WARNING]: Unhandled error in Python interpreter discovery for host 3.123.19.72: Expecting value: line 1 column 1 (char 0)
fatal: [3.123.19.72]: FAILED! => {
    "ansible_facts": {},
    "changed": false,
    "failed_modules": {
        "ansible.legacy.setup": {
            "ansible_facts": {
                "discovered_interpreter_python": "/usr/bin/python3.11"
            },
            "failed": true,
            "module_stderr": "/bin/sh: 1: /usr/bin/python3.11: Permission denied\n",
            "module_stdout": "",
            "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
            "rc": 126,
            "warnings": [
                "Platform linux on host 3.123.19.72 is using the discovered Python interpreter at /usr/bin/python3.11, but future installation of another Python interpreter could change the meaning of that path. See https://docs.ansible.com/ansible-core/2.15/reference_appendices/interpreter_discovery.html for more information."
            ]
        }
    }
}

MSG:

The following modules failed to execute: ansible.legacy.setup
```

Reply:
I did not have such a message on local virtual box machines. I commented out this block so that it would not be executed.